### PR TITLE
feat: reject empty checkout

### PR DESCRIPTION
**Pull request by csandru-plenty**
## Why:

Closes: [AB#131753](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/131753)

## Checklist before requesting a review

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/middleware/reject-empty-checkout.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default defineNuxtRouteMiddleware(async () => {
+  const { getCart, cartIsEmpty } = useCart();
+
+  await getCart();
+  if (!cartIsEmpty.value) return;
+  return navigateTo(useLocalePath()(paths.cart));
```
Comment (maxiroellplenty) [inline apps/web/middleware/reject-empty-checkout.ts]:
```suggestion
  return navigateTo(useLocalePath(paths.cart));
```

Comment (maxiroellplenty) [reply apps/web/middleware/reject-empty-checkout.ts]: should be like this right?

Comment (maxiroellplenty) [reply apps/web/middleware/reject-empty-checkout.ts]: or `useLocalePath().localePath(paths.cart)` ?

**Comment by maxiroellplenty** [comment]
Very clean implementation, love the middleware and the clean isCartEmpty usage, great work 💪

File: apps/web/middleware/reject-empty-checkout.ts (hunk 1 above)
Comment (csandru-plenty) [reply apps/web/middleware/reject-empty-checkout.ts]: Yep, missed that somehow

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**